WordPress/Classes/Utility/WPCrashlytics.m

```diff
@@ -0,0 +1,102 @@
+//
+//  WPCrashlytics.m
+//  WordPress
+//
+//  Created by Diego E. Rey Mendez on 3/26/15.
+//  Copyright (c) 2015 WordPress. All rights reserved.
+//
+
+#import "WPCrashlytics.h"
+
+#import <Crashlytics/Crashlytics.h>
+#import "AccountService.h"
+#import "BlogService.h"
+#import "ContextManager.h"
+#import "WPAccount.h"
+
+@interface WPCrashlytics () <CrashlyticsDelegate>
+@end
```

Comment:
We need it if we want to publish (in the .h) the fact this class implements `CrashlyticsDelegate`.  I usually try to avoid publishing anything in the public header that should not be needed from the outside.